gatewayconfig/bluetooth/characteristics/wifi_connect_characteristic.py

```diff
@@ -18,14 +19,14 @@ class WifiConnectCharacteristic(Characteristic):
     def __init__(self, service):
         self.notifying = False
         Characteristic.__init__(
-                self, constants.WIFI_CONNECT_CHARACTERISTIC_UUID,
-                ["read", "write", "notify"], service)
+            self, constants.WIFI_CONNECT_CHARACTERISTIC_UUID,
```

Comment:
Ideally these auto-lints should go into a separate commit. I also asked @kashifpk not to even do autolinting like this a while ago. My thinking is that unless a linting rule is codified in our workflow [linter](https://github.com/NebraLtd/hm-config/blob/master/.github/linters/.flake8), it is an opinion relative to your environment and not an absolute truth.

@NebraLtd/developers I suggest you make sure your auto linting is consistent with our workflow linting, and/or update the workflow linter too.